integration-test/src/features/concatenate-CRUD-by-file-upload.feature

```diff
@@ -0,0 +1,25 @@
+Feature: concatenate CRUD by file upload
+
+  Background:
+    Given GPD-Upload running
+
+  Scenario: Create and Update same Debt Position by file upload OK
+  // todo
+
+  Scenario: Create and Delete same Debt Position by file upload OK
+  // todo
+
+  Scenario: Create, Update and Delete same Debt Position by file upload OK
+  // todo
+
+  Scenario: N x Create same Debt Position by file upload OK
```

Comment:
`N x` stands for  ?

Reply:
`N` _creation_ by file upload: i.e. `N` files containing debt positions are being created and the files are submitted via POST `N` times
it is not very clear I will write it better